.github/workflows/tests.yml

```diff
@@ -43,3 +43,28 @@ jobs:
         uses: codecov/codecov-action@v3
         with:
           token: ${{ secrets.CODECOV_TOKEN }}
+
+  typesafety:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
```

Comment:
What Python version is this going to use by default?

Reply:
Good point; it should really pick a version. It'll use the value from `.python-version` if that file is present, but if not it'll default to the 'default Python installed on the runner'.